sdk/keyvault/Azure.Security.Keyvault.Keys/src/JsonWebKey.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Azure.Security.Keyvault.Keys
+{
+    public class JsonWebKey
```

Comment:
I always wondered why does it have Json and Web in the name?

Reply:
It's just what it's called in the JOSE standard.  JWK (json web key), JWE (json web encryption), JWS (json web signature), JWT (json web token).  https://tools.ietf.org/html/rfc7517